racunaljka.js

```diff
@@ -0,0 +1,92 @@
+var writes = document.getElementById('screen');
+var currNumber = 0;
+var result = 0;
+
+var buttons = document.querySelector('.allButtons');
+var numberButtons = buttons.querySelectorAll('.number');
+
+for (var i = 0; i <= numberButtons.length - 1; i++) {
+
+    numberButtons[i].onclick = function() {
+
+        var buttonNumber = parseInt(this.innerHTML);
+
+        currNumber = currNumber * 10 + buttonNumber
+
+        writes.value = currNumber;
+
+    }
+};
+
+var operationButtons = buttons.querySelectorAll('.operation');
+
+for (var i = 0; i <= operationButtons.length - 1; i++) {
+    var action = 1;
+    operationButtons[i].onclick = function() {
+
+        var operation = this.innerHTML;
+
+        if (operation === '-') {
+
+            if (action === 1) {
+                result = currNumber;
+                currNumber = 0;
+                writes.value = result;
+                action = 2;
+
+            } else {
+
+                result = result - currNumber;
+                currNumber = 0;
+                writes.value = result;
+
+            }
+        }
+
+        if (operation === '+') {
+
+            if (action === 1) {
+                result = currNumber;
+                currNumber = 0;
+                writes.value = result;
+                action = 2;
+            } else {
+                result = result + currNumber;
```

Comment:
Now it is not enough to remember just *result* and *currNumber*, since you have multiple operations. You need to remember also the previous operation as well. For example:
```
222 + 33 - 5
```
Once you hit minus you have to check the previous operation which is plus and take according action.